Client/Assets/ContextMenu.js

```diff
@@ -74,10 +74,37 @@ addEventListener("touchstart", function (event) {
   var data = {};
   var element = event.target;
 
+  element.addEventListener("touchend", function (event) {
+    cancel();
+
+    // If we're showing the context menu, prevent the page from handling the click event.
+    if (touchHandled) {
+      touchHandled = false;
+      event.preventDefault();
+    }
+  }, true);
+
+  element.addEventListener("touchmove", function (event) {
+    if (longPressTimeout) {
+      var { screenX, screenY } = event.touches[0];
+
+      // Cancel the context menu if finger has moved beyond the maximum allowed distance.
+      if (Math.abs(touchDownX - screenX) > MAX_RADIUS || Math.abs(touchDownY - screenY) > MAX_RADIUS) {
+        cancel();
+      }
+    }
+  });
+
   do {
     if (!data.link && element.localName === "a") {
       data.link = element.href;
 
+      // Don't show the context menu if this is a script link
+      if (element.getAttribute("href") === "#") {
+        data = {};
+        break;
+      }
```

Comment:
This was supposed to be removed from the patch. Unfortunately removing this also breaks the patch with the old behaviour being back. So I am not sure if attaching the event handlers from `touchStart` really made a difference.